resources/servo.css

```diff
@@ -76,13 +76,22 @@ input[type="file"]::before {
   content: "Choose File";
 }
 
-input[type="file"] {
-  background: lightgrey;
+input[type="file"]::before {
   text-align: center;
   vertical-align: middle;
+  padding: 0;
```

Comment:
The removal of padding is to make button-ish border properly being shown, otherwise it'll get cutoff.

Reply:
nvm actually it does nothing to help with the visuals. But the border is pretty apparently cutoff by something.